http://usa.worldfootball.net/all_matches/usa-major-league-soccer-1996/

https://www.pastemagazine.com/articles/2015/03/throwback-thursday-hockey-style-shootouts-in-mls.html

https://www.reddit.com/r/soccer/comments/362y33/back_in_the_days_when_the_mls_had_shootouts/

http://usa.worldfootball.net/report/major-league-soccer-1999-san-jose-earthquakes-d-c-united/

In [27]:
import pandas as pd
data = pd.read_csv('mls_results.csv')
data['score'] = data['selection1_score'].map(lambda x: x if 'n.P' in x else 0)
data_hockey = data.ix[data.score!=0,:]
data_hockey = data_hockey.ix[:, [0,1,5]]

In [31]:
data_hockey['home_score'] = data_hockey['score'].map(lambda x: x[:-5].split(':')[0])
data_hockey['away_score'] = data_hockey['score'].map(lambda x: x[:-5].split(':')[1])
data_hockey['score'] = data_hockey['score'].map(lambda x: x[:-5])

In [37]:
data_hockey.score.value_counts(normalize = True)

2:1    0.255952
1:2    0.190476
2:3    0.142857
3:2    0.125000
0:1    0.113095
1:0    0.077381
4:3    0.041667
3:4    0.041667
5:4    0.005952
4:5    0.005952
Name: score, dtype: float64

In [40]:
2*0.005952

0.011904

In [41]:
data_hockey

selection1_team_1       selection1_team_2 score home_score  \
5               Dallas Burn          San Jose Clash   1:0          1   
10           San Jose Clash     Kansas City Wizards   2:3          2   
12   New England Revolution             D.C. United   2:1          2   
18               MetroStars        Tampa Bay Mutiny   4:3          4   
21              Dallas Burn           Columbus Crew   1:0          1   
28              D.C. United              MetroStars   1:2          1   
39   New England Revolution         Colorado Rapids   3:2          3   
45           San Jose Clash           Columbus Crew   3:2          3   
48          Colorado Rapids             D.C. United   2:3          2   
51   New England Revolution      Los Angeles Galaxy   2:3          2   
53            Columbus Crew     Kansas City Wizards   3:4          3   
56              Dallas Burn         Colorado Rapids   2:1          2   
57   New England Revolution     Kansas City Wizards   2:1          2   
58       Los Angeles Galaxy        Tampa Bay Mutiny   3:2          3   
60               MetroStars             Dallas Burn   1:0          1   
61            Columbus Crew         Colorado Rapids   2:3          2   
63         Tampa Bay Mutiny           Columbus Crew   3:4          3   
68   New England Revolution           Columbus Crew   1:2          1   
77              Dallas Burn             D.C. United   2:1          2   
78           San Jose Clash  New England Revolution   1:2          1   
84            Columbus Crew  New England Revolution   1:2          1   
105         Colorado Rapids      Los Angeles Galaxy   2:3          2   
107  New England Revolution              MetroStars   2:1          2   
109      Los Angeles Galaxy           Columbus Crew   1:2          1   
116     Kansas City Wizards      Los Angeles Galaxy   3:2          3   
119             Dallas Burn           Columbus Crew   2:3          2   
126      Los Angeles Galaxy     Kansas City Wizards   1:2          1   
128          San Jose Clash             Dallas Burn   1:0          1   
136     Kansas City Wizards              MetroStars   3:2          3   
137          San Jose Clash      Los Angeles Galaxy   1:2          1   
..                      ...                     ...   ...        ...   
644              MetroStars        Tampa Bay Mutiny   2:1          2   
645           Columbus Crew          San Jose Clash   4:3          4   
646            Chicago Fire             Dallas Burn   2:1          2   
647        Tampa Bay Mutiny     Kansas City Wizards   1:2          1   
649          San Jose Clash             D.C. United   2:1          2   
654           Columbus Crew             D.C. United   1:2          1   
662            Chicago Fire          San Jose Clash   2:3          2   
667         Colorado Rapids        Tampa Bay Mutiny   2:3          2   
669            Miami Fusion        Tampa Bay Mutiny   3:2          3   
670     Kansas City Wizards          San Jose Clash   3:2          3   
677         Colorado Rapids  New England Revolution   1:0          1   
681             D.C. United        Tampa Bay Mutiny   3:2          3   
687            Chicago Fire            Miami Fusion   1:2          1   
693            Miami Fusion             D.C. United   2:3          2   
696           Columbus Crew        Tampa Bay Mutiny   1:2          1   
697             Dallas Burn            Chicago Fire   4:3          4   
700              MetroStars  New England Revolution   1:2          1   
705          San Jose Clash              MetroStars   2:1          2   
715         Colorado Rapids     Kansas City Wizards   1:0          1   
719            Miami Fusion  New England Revolution   2:1          2   
721            Chicago Fire        Tampa Bay Mutiny   2:1          2   
722             D.C. United              MetroStars   2:1          2   
723     Kansas City Wizards             Dallas Burn   1:2          1   
724  New England Revolution           Columbus Cre

#### Quick Stats

* 168 hockey style shootouts between 1996 and 1999
* Most frequent score: 2-1 (45% of scores)
* Least frequent score: 5-4 (1.2% of scores)